### DISCUSSION TITLE
JW Library Bug

### DISCUSSION THREAD

**Discussion by TheTiger72** [sircharlo/meeting-media-manager#2083] [Q&A] [answered]
Hello, does anyone here have this problem:
When I open JW Library it gets stuck in full screen and I can't move it. To remove it from full screen mode, I have to go to the JW Library settings and deactivate then reactivate the "broadcast on second screen" function. And I have the same problem when I share a screen with Zoom. Thanks

**Comment by sircharlo** [comment]
> Hello, does anyone here have this problem:
> When I open JW Library it gets stuck in full screen and I can't move it. To remove it from full screen mode, I have to go to the JW Library settings and deactivate then reactivate the "broadcast on second screen" function. And I have the same problem when I share a screen with Zoom. Thanks

Is this a bug with JW Library or M³?

**Reply by TheTiger72** [reply]
JW Library

I'm asking my question here because I don't really know where to find help other than here.

**Reply by sircharlo** [reply, marked as answer]
Gotcha. In many cases the local branch office can direct you to get technical support as needed.

**Reply by e-Feitosa** [reply]
@TheTiger72 , In the JW Library access settings and at the end of the instructions you have the support option or access the FAQ at the link https://www.jw.org/en/online-help/jw-library/windows/faqs/.

**Reply by TheTiger72** [reply]
Okay, I'll take a look.
